docs/wiki/auto-deploy.md

```diff
@@ -16,31 +16,22 @@ After a successful code review, the PR is merged. On merge, a GitHub Actions wor
 
 ### 2. Build Site
 
-The workflow builds the exact same container as in local development, so the output build is exactly the same. Unlike local development, this doesn't spin up a development server, instead it saves the site to the filesystem.
+The workflow builds the exact same container as in local development, so the output build is exactly the same. Unlike local development, this doesn't spin up a development server, instead it saves the site to the filesystem of the production server ready for use later on.
```

Comment:
It doesn't magically save it there, it's just that it's run there, so being vague with "the filesystem" is still correct, and probably easier to comprehend. 

```suggestion
The workflow builds the exact same container as in local development, so the output build is exactly the same. Unlike local development, this doesn't spin up a development server, instead it saves the site to the filesystem.
```

Reply:
ah, perhaps I was misreading `deply.yml`, does this line :` script: cd ${{ secrets.PROJECT_PATH }} && docker-compose run wiki build` not just save locally to the right location? Or is there some kind of `mv` command that's not in `deploy.yml`? Either way, will commit the suggestion - thanks!

Reply:
I think the fundamental issue is that steps 2 and 3 are kinda merged, or at least step 2 doesn't exist. 

Step 2 should probably be that the workflow downloads the files to the production server
Step 3 should be the site is built (same "without a dev server" bit)
Step 4 involves restarting the production nginx container (documented [here](https://github.com/selfhostedshow/infra/blob/master/ansible/group_vars/demo.yaml#L133), the `prod` directory in the repo isn't used). Then the stale containers are pruned.

Realise that's actually quite different both from how it is, and your current changes.

Reply:
ahh ha, thank you for the link in Step 4 - that's the bit I was missing. Commit incoming